File: 1470. Shuffle the Array/solution1.java
class Solution {
    public int[] shuffle(int[] nums, int n) {
        var ps = 0;
        var pf = n;
        int [] arr = new int[nums.length];
        var index = 0;
        for(int i = 0 ; i<n;i++){
            
            arr[index++] = nums[ps];
            arr[index] = nums[pf];
            ps++;
            pf++;
            index++;
            
        }
        return arr;
        
    }
}

File: 1470. Shuffle the Array/solution1.py
class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        pf = n
        ps = 0
        lst = []
        for x in range(n):
            lst.append(nums[ps])
            lst.append(nums[pf])
            ps+=1
            pf+=1
        
        return(lst)
